tests/cases/colors.js

```diff
@@ -115,6 +115,28 @@ describe('geo.util.convertColor', function () {
       expect(geo.util.convertColor(undefined)).toBe(undefined);
     });
   });
+  describe('Memoization', function () {
```

Comment:
for some improved reading of the test outputs, I would use Memoization Colors - as we may have more memoization in the future